security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/ConditionalActionsResultDeserializer.java

```diff
@@ -0,0 +1,51 @@
+package com.powsybl.security.json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.security.LimitViolationsResult;
+import com.powsybl.security.PostContingencyComputationStatus;
+import com.powsybl.security.results.NetworkResult;
+import com.powsybl.security.results.OperatorStrategyResult;
+
+import java.io.IOException;
+
+public class ConditionalActionsResultDeserializer extends StdDeserializer<OperatorStrategyResult.ConditionalActionsResult> {
+
+    public ConditionalActionsResultDeserializer() {
+        super(OperatorStrategyResult.ConditionalActionsResult.class);
+    }
+
+    @Override
+    public OperatorStrategyResult.ConditionalActionsResult deserialize(JsonParser parser, DeserializationContext deserializationContext) throws IOException {
+        String conditionalActionsId = null;
```

Comment:
should a builder be better ? I don know

Reply:
I could reuse ConditionalActionsResultBuilder here but I don't think it really brings something more interesting than a simple call to the constructor.